### ISSUE TITLE
Autocomplete popup is displaced

### ISSUE THREAD

**Issue by abbiekressner** [braver/typewriter#12]
The autocomplete popup is displaced to the left of the cursor when the typewriter package is activated. It is as if autocomplete does not know that the typewriter package moved the text to the center of the window.

**Comment by braver** [comment]
I'm not able to reproduce this actually. Can you give me some more info about your setup and perhaps a screenshot?

**Comment by abbiekressner** [comment]
I am running Atom on Ubuntu 14.04 LTS. Here is a screen capture showing what happens:
![screenshot from 2015-12-03 11 08 40](https://cloud.githubusercontent.com/assets/508431/11557715/6e2a191a-99ae-11e5-9397-3b55eadb6696.png)

I am not sure what other information will be useful about my setup since I have not done any programming in Atom at all. Just let me know what info you need. 

Anyway, great work on the plugin! It is otherwise very handy!

**Comment by braver** [comment]
Well, that is weird. Below is what I get with latex highlighting and minimap installed. Your layout is off, the minimap should line up right against the text. Is there some other stylesheet you're using to adjust the layout? Other that that, I  would need more information about the packages you've got installed.

<img width="730" alt="screen shot 2015-12-05 at 13 54 28" src="https://cloud.githubusercontent.com/assets/2543659/11607867/12851258-9b58-11e5-9a59-2e69dc89e928.png">

**Comment by abbiekressner** [comment]
It turns out that I had some lingering settings from before I installed the typewriter package, as copied from [this post](http://rolflekang.com/writing-latex-in-atom/). I restored styles.less, init.coffee, and config.cson to their defaults. Then I changed the scope in the settings of the typewriter package to `text.md,text.tex.latex`. It seems to be working as expected now. Thanks for the quick reply, @braver!
